SPECS-SIGNED/kernel-hci-signed/kernel-hci-signed.spec

```diff
@@ -0,0 +1,521 @@
+#   4. Build this spec
+Source0:        kernel-hci-%{version}-%{release}.%{buildarch}.rpm
+Source1:        vmlinuz-%{uname_r}
+Source2:        sha512hmac-openssl.sh
+BuildRequires:  cpio
+BuildRequires:  openssl
+BuildRequires:  sed
+
+%description
+This package contains the Linux kernel package for HCI with kernel signed with the production key
+
+%package -n     kernel-hci
+Summary:        Linux Kernel for HCI
+Group:          System Environment/Kernel
+Requires:       filesystem
+Requires:       kmod
+Requires(post): coreutils
+Requires(postun): coreutils
+
+%description -n kernel-hci
+The kernel-hci package contains the signed Linux kernel for HCI.
+
+%prep
+
+%build
+mkdir rpm_contents
+pushd rpm_contents
+
+# This spec's whole purpose is to inject the signed kernel binary
+rpm2cpio %{SOURCE0} | cpio -idmv
+cp %{SOURCE1} ./boot/vmlinuz-%{uname_r}
+
+popd
+
+%install
+pushd rpm_contents
+
+# Don't use * wildcard. It does not copy over hidden files in the root folder...
+cp -rp ./. %{buildroot}/
+
+popd
+
+# Recalculate sha512hmac for FIPS
+%{sha512hmac} %{buildroot}/boot/vmlinuz-%{uname_r} | sed -e "s,$RPM_BUILD_ROOT,," > %{buildroot}/boot/.vmlinuz-%{uname_r}.hmac
+cp %{buildroot}/boot/.vmlinuz-%{uname_r}.hmac %{buildroot}/lib/modules/%{uname_r}/.vmlinuz.hmac
+
+%triggerin -n kernel-hci -- initramfs
+mkdir -p %{_localstatedir}/lib/rpm-state/initramfs/pending
+touch %{_localstatedir}/lib/rpm-state/initramfs/pending/%{uname_r}
+echo "initrd generation of kernel %{uname_r} will be triggered later" >&2
+
+%triggerun -n kernel-hci -- initramfs
+rm -rf %{_localstatedir}/lib/rpm-state/initramfs/pending/%{uname_r}
+rm -rf /boot/initrd.img-%{uname_r}
+echo "initrd of kernel %{uname_r} removed" >&2
+
+%postun -n kernel-hci
+if [ ! -e /boot/mariner.cfg ]
+then
+     ls /boot/linux-*.cfg 1> /dev/null 2>&1
+     if [ $? -eq 0 ]
+     then
+          list=`ls -tu /boot/linux-*.cfg | head -n1`
+          test -n "$list" && ln -sf "$list" /boot/mariner.cfg
+     fi
+fi
+
+%post -n kernel-hci
+/sbin/depmod -a %{uname_r}
+ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
+
+%files -n kernel-hci
+%defattr(-,root,root)
+%license COPYING
+/boot/System.map-%{uname_r}
+/boot/config-%{uname_r}
+/boot/vmlinuz-%{uname_r}
+/boot/.vmlinuz-%{uname_r}.hmac
+%config(noreplace) /boot/linux-%{uname_r}.cfg
+%config %{_localstatedir}/lib/initramfs/kernel/%{uname_r}
+%defattr(0644,root,root)
+/lib/modules/%{uname_r}/*
+/lib/modules/%{uname_r}/.vmlinuz.hmac
+%exclude /lib/modules/%{uname_r}/build
+%exclude /lib/modules/%{uname_r}/kernel/drivers/gpu
+%exclude /lib/modules/%{uname_r}/kernel/sound
+%exclude /module_info.ld
+
+%changelog
+* Fri Nov 18 2022 Vince Perri <viperri@microsoft.com> - 5.15.77.1-1
+- Rename package from kernel-signed to kernel-hci-signed.
+- Add ExclusiveArch x86_64 and remove aarch64 build functionality.
+
+* Tue Nov 08 2022 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 5.15.77.1-1
+- Auto-upgrade to 5.15.77.1
```

Comment:
To remove changelog version duplicates and still pass the entangled specs check, I'd suggest removing the servicing account's entry. I think we can remove the rest of the history as well, since this is a new spec.
```suggestion
```